### PR TITLE
FIX: human readable BundeInstance success messages

### PR DESCRIPTION
**Pull request by nsapse**
Modifies bundleinstance_controller to include human readable message in
message field after successful installation of a BundleInstance
resource.

Modifies associated tests to represent that change.

### CODE REVIEW

**Comment by tylerslaton** [comment]
Nice, will approve once the test suite is passing.

**Comment by nsapse** [comment]
running the tests locally I'm getting a lot of sketchy time-out errors on unrelated tests (the test in question passes when focused). Is there something going on network wise?

**Comment by timflannagan** [comment]
@nsapse Can you add the "Closes #281" to your PR description so automation can automatically close this when this PR merges?

**Comment by nsapse** [comment]
Yeah, noticed a lot of these errors are just where empty strings are expected and that's leading to timing out strangely enough. Updating all instances.

> @nsapse Can you add the "Closes #281" to your PR description so automation can automatically close this when this PR merges?

for sure!

File: internal/provisioner/plain/controllers/bundleinstance_controller.go
Comment (exdx) [inline internal/provisioner/plain/controllers/bundleinstance_controller.go:269]: WDYT about making this a bit more helpful and including the name of the bundle that was installed as part of the message? Also, technically the Bundle is "installed" by the bundle controller, the BI controller is creating an instance of that Bundle on-cluster, so I wonder if we can be more precise with our language here. 

For example,
```
fmt.Sprintf(instantiated bundle %s successfully, bi.Spec.BundleName)
```

File: test/e2e/plain_provisioner_test.go
Comment (timflannagan) [inline test/e2e/plain_provisioner_test.go:441]: Let's remove this debug artifact this before merging.

**Review by timflannagan — APPROVED** [review]

**Comment by nsapse** [comment]
Sorry, not up on procedure. Am I good to click the merge button here?

**Comment by timflannagan** [comment]
Yep

**Pull request merged**
